images/other/mindustry.json

```diff
@@ -0,0 +1,82 @@
+{
+  "meta": {
+    "api_version": 2
+  },
+  "uid": "mindustry",
+  "name": "Mindustry",
+  "author": "info@wilfredproject.org",
+  "docker_image": "wilfreddev/java:17",
+  "command": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar server-release.jar config port {{SERVER_PORT}},config name {{image.env.SERVER_NAME}},host {{image.env.MAPNAME}}",
+  "user": "container",
+  "stop_command": "exit",
+  "default_image": true,
+  "config": {
+    "files": []
+  },
+  "installation": {
+    "docker_image": "wilfreddev/debian:latest",
```

Comment:
Is there any particular reason that you need to use Debian instead of something like Alpine (used for most other images in this project simply because it's very lightweight in size)?

Reply:
Well it's the distro i am the most familiar with

Reply:
I see, I can see if I can rewrite it to work in Alpine. the Debian image is way too large for this :slightly_smiling_face:

Reply:
Perfect. While you are rewriting the installer i try to solve the problem with the teamspeak server